src/Mod/Draft/importSVG.py

```diff
@@ -264,44 +265,48 @@ def getsize(length,mode='discard',base=1):
         """parses length values containing number and unit
         with mode 'discard': extracts a number from the given string (removes unit suffixes)
         with mode 'tuple': return number and unit as a tuple
-        with mode 'css': convert the unit to px assuming 90dpi
-        with mode 'mm': convert the unit to millimeter assuming 90dpi"""
-        tomm={
-                '' : 25.4/90, #default
-                'px' : 25.4/90,
-                'pt' : 1.25*25.4/90,
-                'pc' : 15*25.4/90,
-                'mm' : 1.0,
-                'cm' : 10.0,
-                'in' : 25.4,
-                'em':  15*2.54/90, #arbitrarily chosen; has to depend on font size
-                'ex': 10*2.54/90, #arbitrarily chosen; has to depend on font size
-
-                '%': 100 #arbitrarily chosen; has to depend on vieport size or (for filling patterns) on bounding box
-                }
-        topx={
-                '' : 1.0, #default
-                'px' : 1.0,
-                'pt' : 1.25,
-                'pc' : 15,
-                'mm' : 90.0/25.4,
-                'cm' : 90.0/254.0,
-                'in' : 90,
-                'em':  15, #arbitrarily chosen; has to depend on font size
-                'ex':  10, #arbitrarily chosen; has to depend on font size
-
-                '%': 100 #arbitrarily chosen; has to depend on vieport size or (for filling patterns) on bounding box
-                }
+        with mode 'css90.0': convert the unit to px assuming 90dpi
+        with mode 'css96.0': convert the unit to px assuming 96dpi
+        with mode 'mm90.0': convert the unit to millimeter assuming 90dpi
+        with mode 'mm96.0': convert the unit to millimeter assuming 96dpi"""
+        if mode == 'mm96.0' or mode == 'mm90.0':
+            tomm={
+                    '' : 25.4/float(mode[-4]), #default
+                    'px' : 25.4/float(mode[-4]),
+                    'pt' : 1.25*25.4/float(mode[-4]),
+                    'pc' : 15*25.4/float(mode[-4]),
+                    'mm' : 1.0,
+                    'cm' : 10.0,
+                    'in' : 25.4,
+                    'em':  15*2.54/float(mode[-4]), #arbitrarily chosen; has to depend on font size
+                    'ex': 10*2.54/float(mode[-4]), #arbitrarily chosen; has to depend on font size
+
+                    '%': 100 #arbitrarily chosen; has to depend on vieport size or (for filling patterns) on bounding box
+                    }
+        if mode == 'css96.0' or mode == 'css90.0':
+            topx={
+                    '' : 1.0, #default
+                    'px' : 1.0,
+                    'pt' : 1.25,
+                    'pc' : 15,
+                    'mm' : float(mode[-4])/25.4,
```

Comment:
Why floats? Aren't they extremely inaccurate in some situations and better suited for quick calculations, where accuracy is not important?

Reply:
I've been checking the distances to 6 decimal places and I thought I was just replacing hard coded numbers with the same. What do you suggest I use instead, the Draft.precision function?